library/init/meta/interactive_base.lean

```diff
@@ -59,6 +59,9 @@ meta def brackets (l r : string) (p : parser α) := tk l *> p <* tk r
 
 meta def list_of (p : parser α) := brackets "[" "]" $ sep_by (skip_info (tk ",")) p
 
+precedence `⊢` : 0
```

Comment:
Otherwise `rw at h1 ⊢` doesn't parse correctly. The scanner rejects it before it has a chance to be parsed.